modules/nw-routeadvertisements-about.adoc

```diff
@@ -0,0 +1,93 @@
+
+////
+Terminology -
+Cluster network routes: Both pod network routes and/or EgressIP routes
+Pod network routes: Routes to pod IPs
+EgressIP routes: Routes to EgressIPs
+////
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-routeadvertisements-about_{context}"]
+= Advertise cluster network routes with Border Gateway Protocol
+
+With route advertisements enabled, the OVN-Kubernetes network plugin supports advertising cluster network routes, including EgressIPs, to the provider network, and importing routes from the provider network to the cluster network.
+
+For example, routes are dynamically discovered, so you no longer need to manually configure routes on each node. Routing for cluster user-defined (CUDN) networks can be announced to the provider network, avoiding manual configuration steps.
+
+[id="supported-platforms_{context}"]
+== Supported platforms
+
+Advertising routes with BGP is supported on the following infrastructure types:
+
+- Bare-metal
+//- {vmw-full} on-premise
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+To use route advertisements, you must have configured BGP for your network infrastructure. Outages or misconfigurations of your network infrastructure might cause disruptions to your cluster network.
+
+[id="cluster-network-operator_{context}"]
+== Cluster Network Operator configuration
+
+The Cluster Network Operator (CNO) API exposes several fields to configure route advertisements:
+
+- `spec.additionalRoutingCapabilities.providers`: Specifies an additional routing provider, which is required to advertise routes. The only supported value is `FRR`, which enables deployment of the FRR-K8S daemon for the cluster. When enabled, the FRR-K8S daemon is deployed on all nodes.
+- `spec.defaultNetwork.ovnKubernetesConfig.routeAdvertisements`: Enables route advertisements for the default cluster network and CUDN networks. The `spec.additionalRoutingCapabilities` field must be set to `FRR` to enable this feature.
+
+[id="compatibility-with-other-networking-features_{context}"]
+== Compatibility with other networking features
+
+Route advertisements support the following {product-title} Networking features:
+
+Multiple external gateways::
+--
+When used with multiple external gateways (MEG) as implemented with the `AdminPolicyBasedExternalRoute` CR, the following behavior is undefined:
+
+- When BGP discovers routes to next hops, there can overlap with gateways configured with MEG or AdminPolicyBasedExternalRoute CRs. The behavior for this is undefined.
+--
+
+EgressIPs::
+--
+Supports the use and advertisement of EgressIPs. The node where an egress IP address resides advertises the EgressIP. An egress IP address must be on the same layer 2 network subnet as the egress node. The following limitations apply:
+
+- Advertising EgressIPs from a user-defined network (CUDN) operating in layer 2 mode are not supported.
+- Advertising the pod network and secondary interface EgressIPs at the same time is not supported.
```

Comment:
This sentence is not accurate, we can advertise pod network of default network or CUDN.    It is Secondary egressIP does not work when default network is advertised, according to page 24 of https://docs.google.com/presentation/d/1I3vNQDAjGHV8NfPWWumeZQAmlsxChYFWJ0O4BKS2W64/edit?slide=id.g35774536bc9_0_5197#slide=id.g35774536bc9_0_5197, so customer should not try to use secondary egressIP with default network is advertised.